test/uk/gov/hmrc/driverinspectionnotificationfrontend/controllers/SearchResultControllerSpec.scala

```diff
@@ -732,6 +734,117 @@ class SearchResultControllerSpec extends ControllerBaseSpec {
         contentAsString(result) should include("Find more information on")
         contentAsString(result) should include("if your vehicle could be exempt from attending Sevington.")
       }
+
+      "show content as expected when there is only defra transit inspection type" in new SetUp {
+        val gmrId = "gmrId"
+
+        val reportLocations: List[ReportLocations] = List(ReportLocations(inspectionTypeId = "18", locationIds = List.empty))
+        val inspectionTypeWithLocations = List(Right(InspectionTypeWithLocations(InspectionType("18", "DEFRA_TRANSIT"), List.empty)))
+
+        when(mockGmsService.getInspectionStatus(argEq(gmrId))(any()))
+          .thenReturn(
+            EitherT.rightT[Future, GmrErrors](
+              inspectionResponse(
+                direction = UK_INBOUND,
+                inspectionStatus = InspectionStatus.InspectionRequired,
+                reportToLocations = Some(reportLocations)
+              )
+            )
+          )
+
+        when(mockReferenceDataService.getInspectionData(argEq(reportLocations))(any()))
+          .thenReturn(inspectionTypeWithLocations)
+
+        val result = controller.result(gmrId, checkedStatusAgain = false)(FakeRequest())
+        val content: String = contentAsString(result)
+
+        status(result) shouldBe 200
+        content          should include("The goods you are moving require an inspection")
+        content          should include("What to do next")
+        content          should include("For your DEFRA transit inspection")
+        content should include(
+          "ak to your haulier or importer to find out which Border Control Post (BCP) or other inspection location you need to attend on arrival."
```

Comment:
typo in `ak`

Reply:
oh, partial match may be, I think it is Speak

Reply:
fixed